observability/logging/logging-6.0/log6x-configuring-lokistack-otlp.adoc

```diff
@@ -0,0 +1,146 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="log6x-configuring-lokistack-otlp"]
+= OTLP data ingestion in Loki
+include::_attributes/common-attributes.adoc[]
+:context: log6x-configuring-lokistack-otlp
+
+toc::[]
+
+Loki 3.0 introduces an API endpoint using the OpenTelemetry Protocol (OTLP), providing a new way to ingest log entries into Loki. This endpoint complements the standard Push API, which is available in Loki from its initial release. Since OTLP is a standardized format not specifically designed for Loki, it requires additional configuration on Loki's side to map OpenTelemetry's data format to Loki's data model. OTLP lacks concepts such as *stream labels* or *structured metadata*. Instead, OTLP provides metadata about log entries as *attributes*, grouped into three categories:
```

Comment:
Do *stream labels* and *structured metadata* really need to be emphasized? Usually we would use italics for emphasis, but should generally avoid (per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference)):

> Use of underscores for general emphasis is allowed but should only be used very sparingly. Let the writing, instead of font usage, create the emphasis wherever possible.

Italics can be fine for a "first term/instance" style usage, if it's something that's going to be repeated.

Reply:
Lets use italics here. 
![Screenshot from 2024-11-08 17-29-26](https://github.com/user-attachments/assets/c05d61bc-ffbe-4e2c-b0a0-5134301cc0fc)